tests/test_influxdb.py

```diff
@@ -1,17 +1,22 @@
+import os
+
 from fqn_decorators import get_fqn
 from influxdb.influxdb08.client import InfluxDBClientError
+
 from tests.conftest import Dummy, go
 from tests.test_base_backend import TestBaseBackend
 from time_execution import settings
 from time_execution.backends.influxdb import InfluxBackend
 
+INFLUX_HOST = os.getenv("INFLUX_HOST")
```

Comment:
Perhaps, not relevant for this PR. But we should maybe also drop InfluxDB support.

Reply:
but there are so many people using it all over the world with `timeexecution` ;)

Reply:
what about kafka? it was never tested on prod. should we drop it as well?